src/main/java/core/basesyntax/calculation/CalculationStatistics.java

```diff
@@ -0,0 +1,25 @@
+package core.basesyntax.calculation;
+
+import java.util.ArrayList;
+
+public class CalculationStatistics {
+    private static final String BUY_OPERATION = "buy";
+    private static final String SUPPLY_OPERATION = "supply";
+
+    public static String getNumber(ArrayList<String> str) {
+        int supplyOperation = 0;
+        int buyOperation = 0;
+        for (String strO : str) {
+            if (strO.trim().startsWith(SUPPLY_OPERATION)) {
+                supplyOperation += Integer.parseInt(strO.substring(strO.indexOf(',') + 1));
+            } else if (strO.trim().startsWith(BUY_OPERATION)) {
+                buyOperation += Integer.parseInt(strO.substring(strO.indexOf(',') + 1));
+            } else {
+                break;
+            }
+        }
+        return "supply," + supplyOperation + System.lineSeparator()
```

Comment:
use StringBuilder instead of String concatenation

Reply:
> use StringBuilder instead of String concatenation
Ther`s no loop, so what a reason use StringBuilder?

Reply:
because it makes your program faster

Reply:
According to oracle documents there's no difference in this case and cod is shorter if we useing +  it's easier to read 
https://docs.oracle.com/javase/specs/jls/se8/html/jls-15.html#jls-15.18.1